ocw_data_parser/utils.py

```diff
@@ -280,3 +287,52 @@ def parse_all(  # pylint: disable=too-many-arguments, too-many-locals
             parser.export_parsed_json(
                 s3_links=s3_links, upload_parsed_json=perform_upload
             )
+
+
+def convert_to_vtt(loaded_json):
+    """
+    Convert a json file with srt captions data to vtt format
+
+    Args:
+        loaded_json (dict): The content of JSON file
+
+    Returns:
+        dict: copy of the json file with the _datafield_file converted to vtt format
+    """
+    if loaded_json["id"].endswith(".vtt"):
+        return None
+    new_json = dict(loaded_json)
+
+    new_json["id"] = update_srt_to_vtt(loaded_json["id"])
+    new_json["technical_location"] = update_srt_to_vtt(
+        loaded_json["technical_location"]
+    )
+    new_json["_uid"] = uuid.uuid1().hex
```

Comment:
Thanks, this shows up in the output now. However I realized this produces random output which can make it hard to verify the output of changes to ocw-data-parser. Can we add a prefix or suffix to the existing `_uid` instead?

Reply:
Can the prefix be "vtt"? or does it need to be random?

Reply:
vtt is fine